helm-chart/kuberay-operator/values.yaml

```diff
@@ -78,10 +78,9 @@ batchScheduler:
   # Deprecated. This option will be removed in the future.
   # Note, for backwards compatibility. When it sets to true, it enables volcano scheduler integration.
   enabled: false
-  # Name of the scheduler, currently supported "default", "volcano" and "yunikorn",
-  # set the customized scheduler name, e.g "volcano" or "yunikorn", do not set
+  # Set the customized scheduler name, supported values are "volcano" or "yunikorn", do not set
   # "batchScheduler.enabled=true" at the same time as it will override this option.
-  name: default
+  name: ""
```

Comment:
Should we update `helm-chart/kuberay-operator/templates/deployment.yaml` such that --batch-scheduler flag is not set if name is empty?

Reply:
It is already there: https://github.com/ray-project/kuberay/blob/0e1c248eaab9176251c14d5dbae5011d57e5c154/helm-chart/kuberay-operator/templates/deployment.yaml#L60-L67.

Reply:
Also did some tests:

1. Default install
```
helm install kuberay-operator ../helm-chart/kuberay-operator
```
the args:
```
   Args:
      --feature-gates=RayClusterStatusConditions=false
      --enable-leader-election=true
```

2. Legacy option
```
helm install kuberay-operator --set batchScheduler.enabled=true ../helm-chart/kuberay-operator 
```
the args in the deployment:
```
    Args:
      --feature-gates=RayClusterStatusConditions=false
      --enable-batch-scheduler
      --enable-leader-election=true
```

3. New option
```
helm install kuberay-operator  --set batchScheduler.name=yunikorn ../helm-chart/kuberay-operator
```
the args

```
   Args:
      --feature-gates=RayClusterStatusConditions=false
      --batch-scheduler=yunikor
      --enable-leader-election=true
```